tests/network/primary_pod_network.go

```diff
@@ -88,13 +88,13 @@ var _ = SIGDescribe("[Serial]Primary Pod Network", func() {
 				var vmi *v1.VirtualMachineInstance
 
 				BeforeEach(func() {
-					tmpVmi, err := newFedoraWithGuestAgent()
+					var err error
```

Comment:
this file is related to json yaml great finding ?

Reply:
To be really sure that setting ipv6 using cloud-init is working fine I have refactor just one test to prevent it to be done with `virtctl console`.

Reply:
we dont check in this test that there is ipv6 in the VM do we (we validate just the virt-launcher but not the masquerade) ?
(even that its done in cloud-init for fedora IIRC/IIUC)

Reply:
This is all the validations we do https://github.com/kubevirt/kubevirt/blob/cf891f6135fb673b1623b72ae5e62bab3f7ea611/tests/libnet/validation.go#L31

Reply:
You right I have remove the fix and run the tests and is passing so it does not exercise what cloud-init is doing.

Reply:
see please CreateDefaultCloudInitNetworkData
you might find the tests that use it interesting ?

Reply:
> You right I have remove the fix and run the tests and is passing so it does not exercise what cloud-init is doing.

I guessing we have a production bug.

Reply:
you can try to remove configureIPv6OnVMI from fedora and see if tests that should check the VM ipv6 are passing maybe?
as its a fallback that can mask stuff indeed

Reply:
I am going to migrate https://github.com/kubevirt/kubevirt/blob/master/tests/vmi_multus_test.go#L490 so we test it there.